src/components/ChoiceButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import {IChoice} from "./../events/core";
+
+interface IProps {
+    choice: IChoice;
+    makeChoice: Function;

```

Comment:
it's better to turn this into ```makeChoice: IChoice => void;```

Reply:
This didn't quite compile, but I used
```
makeChoice(choice: IChoice): void;
```
and it worked.